### PR TITLE
fix(curriculum): clarify syntax in directions for RPG game step 65

### LINKED ISSUE #53427: Update step 65 instructions to make it clear on correct syntax
### Describe the Issue

There is a lot of confusion on the correct syntax for step 65
https://forum.freecodecamp.org/search?q=step%2065%20%23javascript%20Role%20Playing%20Game

A lot of campers are struglling to get to here 

```js
location["button text"][0]
```
instead, we have these types of answers

```js
location["button text"[0]]
location["button text[0]"]
```

We should simplify the directions and show the syntax.

## updated directions 

```md
# --description--

`location["button text"]` is an array with three elements. Change the `button1.innerText` assignment to be `location["button text"][0]` which represents the first element of the array.
```

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-basic-javascript-by-building-a-role-playing-game/step-65

### Your code

see explanation above

### Expected behavior

see explanation above

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Macky151203**
Checklist:

- [ x ] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [ x ] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [ x ] My pull request targets the `main` branch of freeCodeCamp.
- [ x ] I have tested these changes either locally on my machine, or GitPod.

Closes #53427

Changed the value of button.innertext in the mentioned file

### CODE REVIEW

**Comment by jdwilkin4** [comment]
@Macky151203 

Please make sure to update your PR to include the issue this is linked to 

https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8b9770050d217d2247801.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ button3.onclick = fightDragon;
 
 --fcc-editable-region--
 function update(location) {
-  button1.innerText = location["button text"];
+  button1.innerText = location["button text"][0];
```
Comment (jdwilkin4) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8b9770050d217d2247801.md:152]: These changes are incorrect.

Please revert back to the original.

The issue wants you to update the description text not the seed code here.

Please make the changes outlined in the issue here

https://github.com/freeCodeCamp/freeCodeCamp/issues/53426

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8b9770050d217d2247801.md (hunk 1 above)
Comment (Macky151203) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8b9770050d217d2247801.md]: Changed the description now

**Comment by jdwilkin4** [comment]
@Macky151203 

Please address this issue too

https://github.com/freeCodeCamp/freeCodeCamp/pull/53431#issuecomment-1915519022

It is important to link the issue you are working on with the PR so maintainers know what these changes are in reference to.
That way, they can effectively review your PR because they will have all of the context for these changes

**Review by jdwilkin4 — APPROVED** [review]

**Review by JoyShaheb — APPROVED** [review]

**Pull request merged**